src/sstruct_ls/HYPRE_sstruct_split.c

```diff
@@ -185,10 +185,17 @@ HYPRE_SStructSplitSetup( HYPRE_SStructSolver solver,
    nparts = hypre_SStructMatrixNParts(A);
    nvars = hypre_TAlloc(HYPRE_Int,  nparts, HYPRE_MEMORY_HOST);
    smatvec_data    = hypre_TAlloc(void ***,  nparts, HYPRE_MEMORY_HOST);
-   ssolver_solve   = (HYPRE_Int (***)()) hypre_MAlloc((sizeof(HYPRE_Int (**)()) * nparts),
-                                                      HYPRE_MEMORY_HOST);
-   ssolver_destroy = (HYPRE_Int (***)()) hypre_MAlloc((sizeof(HYPRE_Int (**)()) * nparts),
-                                                      HYPRE_MEMORY_HOST);
+
+   // RL: TODO TAlloc?
+   ssolver_solve   = (HYPRE_Int (***)()) hypre_MAlloc((sizeof(HYPRE_Int (**)()) * nparts), HYPRE_MEMORY_HOST);
```

Comment:
What is the reason for moving from `hypre_MAlloc` to `hypre_TAlloc` in some places? Just curious

Reply:
Since memory tracker can only track the `hypre_T` macros.

Reply:
I am not familiar with the `HYPRE_Int (**)()` and `HYPRE_Int (***)()` syntax. Is this some C style? Because of it, I don't know how to write these mallocs with `TAlloc`.

Reply:
Maybe it's for function pointers? I'm not that familiar either, but I guess we can take a look at this later

Reply:
Function pointer syntax is strange, but we can definitely save this for later.  Thanks!